scripts/linkerd/benchmark-multi.sh

```diff
@@ -17,25 +17,28 @@ KUBECONFIG=$(print_kubeconfig_path "$asset_dir")
     exit 1; }
 export KUBECONFIG
 
-nr_apps="10"
```

Comment:
Ditto here regarding defaults.

Reply:
I've removed the updated defaults from this PR and will issue a separate PR for that. Thank you for calling this out!.